k8s-oci.tf

```diff
@@ -471,9 +470,15 @@ module "k8smaster-public-lb" {
   shape                     = "${var.k8sMasterLBShape}"
 }
 
+module "nginx-lb" {
+  source        = "./network/loadbalancers/nginx"
```

Comment:
Similarly, lets move / rename this to `./network/loadbalancers/software/reverse-proxy/k8smaster`